bootloader/src/pyi_pythonlib.c

```diff
@@ -88,12 +88,13 @@ pyi_pylib_load(ARCHIVE_STATUS *status)
       strncpy(dllname, status->cookie.pylibname, 64);
     }
 #else
-    len = 0;
     strncpy(dllname, status->cookie.pylibname, 64);
 #endif
 
     if (len >= 64 || dllname[64-1] != '\0') {
-        FATALERROR("DLL name length exceeds buffer\n");
+        dllname[64-1] = 0;
```

Comment:
dllname is a array of char, right? So shouldn't this be `\0`? But when not outputting `dllname` below, this is no necessary at all.

Reply:
iirc - the compier knows it is a "char" type assignment, but syntactically typing it as a char is better.

Maybe "dllname[64-1] = (char) 0;" is even more declarative.

Reply:
I'm curious: Why not simply `'\0'`, which is what is tested above?

I would not trust the compiler casting the type - some might issue a warning about it.

Reply:
removed. Will not try to print dllname as part of error message.